odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1304,6 +1304,15 @@ def _postprocess_tag_field(self, node, name_manager, node_info):
                     for arch, _view in self._get_x2many_missing_view_archs(field, node, node_info):
                         node.append(arch)
 
+                if (
+                    field.relational
+                    and not node.get('options')
```

Comment:
Wouldn't it be nice to have default values instead of having to override everything?
A bit like done for the module manifests